enzyme/benchmarks/CMakeLists.txt

```diff
@@ -46,3 +46,5 @@ add_lit_testsuite(bench-enzyme "Running enzyme benchmarks tests"
 set_target_properties(bench-enzyme PROPERTIES FOLDER "bench Tests")
 
```

Comment:
Can the file structure here be changed to have only one copy of the benchmark data?